notificationdemo/pom.xml

```diff
@@ -123,6 +126,44 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.rat</groupId>
+                <artifactId>apache-rat-plugin</artifactId>
+                <configuration>
+                    <useDefaultExcludes>true</useDefaultExcludes>
+                    <excludes>
+                        <!--Android Studio -->
+                        <exclude>source/android/gradlew</exclude>
```

Comment:
this exclusions will be common for all android projects. Maybe we can remove them to main pom to avoid code duplication?